org.eclipse.xtext.xbase.testing/src/org/eclipse/xtext/xbase/testing/OnTheFlyJavaCompiler2.xtend

```diff
@@ -114,7 +114,7 @@ class OnTheFlyJavaCompiler2 {
 				}
 			}
 		''')
-		return clazz.newInstance as Functions.Function0<RT>
+		return clazz.constructor.newInstance as Functions.Function0<RT>
```

Comment:
why not declaredConstructor?

Reply:
cause i could not force push ;)
should be now declared one